common/app/common/Logback/Logstash.scala

```diff
@@ -29,7 +30,8 @@ object Logstash {
     "app" -> Configuration.environment.projectName,
     "stage" -> Configuration.environment.stage.toUpperCase,
     "build" -> ManifestData.build,
-    "revision" -> ManifestData.revision
+    "revision" -> ManifestData.revision,
+    "ec2_instance" -> Option(EC2MetadataUtils.getInstanceId).getOrElse("Not running on ec2")
```

Comment:
Good point. This is only executed once when the app starts and the logger is setup so it is not a big deal here. However I am happy to improve if you have a better idea